beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/repository/HousekeepingPathRepository.java

```diff
@@ -36,4 +37,9 @@
       + "and p.modifiedTimestamp <= :instant order by p.modifiedTimestamp")
   Page<HousekeepingPath> findRecordsForCleanupByModifiedTimestamp(@Param("instant") LocalDateTime instant,
       Pageable pageable);
+
+  @Modifying
+  @Query(value = "delete from HousekeepingPath p where p.cleanupTimestamp < :instant "
```

Comment:
Let's order it by `order by p.cleandupTimestamp` so we are deleting rows in order. I found it useful to see the progress while debugging.

Reply:
ok

Reply:
tried to add the order by, but the tests fail cause apparently the queries cannot be validated 🤔

Reply:
hmm.. how is it working with the getPathsToBeCleaned() query ?

Reply:
Discussed offline. This is not a blocker. So, I will go ahead and approve this PR.